geoserverexplorer/qgis/sldadapter.py

```diff
@@ -90,7 +90,15 @@ def adaptQgsToGs(sld, layer):
         sld = sld.replace("<sld:WellKnownName>%s</sld:WellKnownName>" % key,
                       "<sld:WellKnownName>%s</sld:WellKnownName>" % value)
 
-
+    fontmarkers = re.findall('<sld:OnlineResource xlink:type="simple" xlink:href=".*?"/> <sld:Format>ttf</sld:Format> <sld:MarkIndex>.*?</sld:MarkIndex>', sld)
```

Comment:
thank you for the contribution, just a note, the order of tag parameters can't be fixed.
regular expression should be wrote considering that the order of "xlink:type" and "xlink:href" can differ.